lighthouse-core/report/html/report-styles.css

```diff
@@ -320,7 +335,7 @@
 }
 
 .lh-audit__description, .lh-audit__stackpack {
-  --inner-audit-right-padding: calc(var(--text-indent) + 2px);
+  --inner-audit-right-padding: var(--stackpack-horizontal-padding);
```

Comment:
elsewhere we used `margin-right` which seemed to align with the convention, flip here too to match property name?

Reply:
for sure